ios/RNSScreenViewEvent.mm

```diff
@@ -21,6 +21,7 @@ - (instancetype)initWithEventName:(NSString *)eventName
   if ((self = [super init])) {
     _eventName = [eventName copy];
     _viewTag = reactTag;
+
```

Comment:
why blank line here?

Reply:
As I stated in this commit message: "[random change to trigger the CI](https://github.com/software-mansion/react-native-screens/pull/1503/commits/00c1e6e31b5e695eb7e83a3d6c792251e46017e0)" - so that I don't have to trigger workflows manually. It will be obviously deleted when the job is done (I will just remove this commit).